x-pack/plugins/infra/public/components/logging/log_text_stream/scrollable_log_text_stream_view.tsx

```diff
@@ -246,6 +271,39 @@ class ScrollableLogTextStreamViewClass extends React.PureComponent<
 
 export const ScrollableLogTextStreamView = injectI18n(ScrollableLogTextStreamViewClass);
 
+/**
+ * This function-as-child component calculates the column widths based on the
+ * given configuration. It depends on the `CharacterDimensionsProbe` it returns
+ * being rendered so it can measure the monospace character size.
+ *
+ * If the above component wasn't a class component, this would have been
```

Comment:
Thanks for the note 👍

Reply:
It's on my cleanup list :crossed_fingers: